### DISCUSSION TITLE
Option to install avahi-daemon automatically

### DISCUSSION THREAD

**Discussion by pgodzwa** [tteck/Proxmox#2848] [Ideas]
Would it be possible for you to add avahi-daemon package to be installed automatically or on choice via advanced options?
I mean it is easy to install it manually after creating LXC/VM, but I think it could be useful as it allows to participate in mDNS protocol, then allowing to use shortnames <vm/lxc_name>.local to access services instead of using IP

Thanks in advance!

**Comment by dada051** [comment]
Maybe I'm doing something wrong, but I always need to install avahi-utils after first LXC boot to be able to SSH into the newly created LXC with its name. It's not so complicated to SSH with the IP when it has been set as static, but its more work when set as dhcp.
So adding the option to add avahi-utils can be great as it makes the full process more fluent (I understand that it's not a necessary package that had to be added OOB). So +1 for this discussion :)
